features/unhandled_exception.feature

```diff
@@ -2,17 +2,17 @@ Feature: NSException handling
 
 Scenario: Uncaught NSException is raised
     When I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And I set environment variable "EVENT_TYPE" to "NSException"
+    And I set environment variable "EVENT_TYPE" to "HandledExceptionScenario"
```

Comment:
There should be a scenario to handle an NSException being raised. Its not on the test list since it was written already for the branch. The `Scenario:` title should also always match the contents.